visualizations/store-map-viz/components/Region.tsx

```diff
@@ -34,7 +56,7 @@ const Region = ({ region, location, tooltipConfig, defaultHeader }) => {
   };
 
   return (
-    <GeoJSON key={location.status} data={region} style={style} onClick={handleRegionClick}>
+    <GeoJSON key={key+"-"+location.value+"-"+style.fillColor} data={region} style={style} onClick={handleRegionClick}>
```

Comment:
I changed this so that if the color changes it re-renders, is there a better way?